_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Version 1
+---
+
+We are happy to release version 1 of MDAnalysis!
+
+
+MDAnalysis 1.x is a stable legacy platform that supports 
+
+-  Python 2.7 (on Linux and MacOS only), 
```

Comment:
If we do not have conda packages for 2.7 then we need to say so somewhere, perhaps link to issue https://github.com/MDAnalysis/mdanalysis/issues/3105